go.mod

```diff
@@ -155,10 +155,14 @@ replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alp
 replace github.com/99designs/keyring => github.com/crypto-org-chain/keyring v1.1.6-fixes
 
 // TODO: remove when middleware will be implemented
-replace github.com/cosmos/ibc-go/v2 => github.com/crypto-org-chain/ibc-go/v2 v2.0.1-hooks
+replace github.com/cosmos/ibc-go/v3 => github.com/yihuang/ibc-go/v3 v3.0.0-hooks-1
```

Comment:
TODO, update to crypto-org-chain later.

Reply:
I have created a tag in crypto-org-chain for you to use it